python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -86,9 +86,12 @@ def __deepcopy__(self, memo=None):
         cls = self.__class__
         obj = cls()
         this_dict = copy(self.__dict__)
-        for elem in this_dict.keys():
+        global do_not_copy
```

Comment:
Good question, unfortunately I've no answer. I plan to revert this method anyway so it'll drop then.